README.md

```diff
@@ -128,86 +91,24 @@ You can dynamically decide which transport to use, basing e.g. on the user-agent
 
 ### Basic usage
 
-Sessions are typed; the `T` type parameter in `SessionManager[T]` determines what data is stored in the session. 
-Basic types like `String`, `Int`, `Long`, `Float`, `Double` and `Map[String, String]` are supported out-of-the box. 
-Support for other types can be added by providing an implicit `SessionSerializer[T, String]`. For case classes, it's most 
-convenient to use a `MultiValueSessionSerializer[T]` which should convert the instance into a `String -> String` map 
-(nested types are not supported on purpose, as session data should be small & simple).
-
-Here we are creating a manager where the session content will be a single `Long` number:
+Sessions are typed. The `T` type parameter in `SessionManager[T]` (or `SessionManager<T>`) determines what data is stored in the session. 
+Basic types like `String`, `Int`, `Long`, `Float`, `Double` and `Map[String, String]` (`Map<String, String>`) are supported out-of-the box. 
+Support for other types can be added by providing an implicit `SessionSerializer[T, String]` (`SessionSerializer<T, String>`). For case classes, it's most 
```

Comment:
`implicit` refers to Scala. Is it possible to write this down in a language-agnostic fashion? ;)